common/src/main/java/com/microsoft/identity/common/adal/internal/AuthenticationConstants.java

```diff
@@ -1388,46 +1451,56 @@
          */
         public static final String AUTHORITY = "microsoft.identity.broker";
 
+        private static final String VERSION_1 = ONE_POINT_ZERO;
+        private static final String VERSION_5 = FIVE_POINT_ZERO;
+        private static final String VERSION_6 = "6.0";
```

Comment:
nit: Why can this be "6.0"
(Actually, why do we need stuffs like "ONE_POINT_ZERO")?

Reply:
Because if I put more than three copies of "1.0" in the file, PMD fails the build.  I don't want to change our ruleset right now.